src/decaf-qt/src/mainwindow.cpp

```diff
@@ -13,6 +13,7 @@
 #include <QFileDialog>
 #include <QInputDialog>
 #include <QSettings>
+#include <QFileInfo.h>
```

Comment:
```suggestion
#include <QFileInfo>
```
this breaks linux CI